velox/common/memory/ByteStream.h

```diff
@@ -96,10 +188,6 @@ class OStreamOutputStream : public OutputStream {
 /// seeking back to start to write a length header.
 class ByteStream {
```

Comment:
I assume we will rename this to ByteOutputStream in followup? Thanks!

Reply:
Yes, that's the plan. Just trying to limit the number of changes in a single PR.